code/modules/mob/living/carbon/human/human.dm

```diff
@@ -213,6 +213,10 @@
 	if(legcuffed)
 		dat += "<tr><td><A href='?src=\ref[src];item=[slot_legcuffed]'>Legcuffed</A></td></tr>"
 
+	for(var/obj/item/bodypart/L in bodyparts)
+		for(var/obj/item/I in L.embedded_objects)

```

Comment:
embedded_objects could be almost anything though, so I think it's okay in this case